partest/src/main/scala/scala/tools/partest/scalajs/ScalaJSPartest.scala

```diff
@@ -190,15 +192,12 @@ class ScalaJSSBTRunner(
   // Partests take at least 5h. We double, just to be sure. (default is 4 hours)
   sys.props("partest.timeout") = "10 hours"
 
-  // Set showDiff on global UI module
-  if (options.showDiff)
-    NestUI.setDiffOnFail()
```

Comment:
[It went here](https://github.com/scala-js/scala-js/pull/2468/files#diff-a6531f0ed7cf3feadd0280492d634f17R40) due to a change in design of the `NestUI`.